.gitlab/lassen-jobs.yml

```diff
@@ -75,18 +75,18 @@ gcc_8_3_1_cuda:
 
 pgi_19_10_cuda:
   variables:
-    SPEC: "+fortran+cuda %pgi@19.10 ^cuda@10.1.168"
+    SPEC: "+fortran+cuda %pgi@19.10 ^cuda@11.0.2"
```

Comment:
If you switch this back to cuda 10 it should pass.

Reply:
(was already in progress hehe i ran the tests and it passes locally at least! fingers crossed lol)